ScoutHelper/Data/Siren.json

```diff
@@ -1,18 +1,12 @@
 ﻿{
 	"HW": {
 		"mob order": [
-			"lyuba",
-			"mirka",
-			"enkelados",
-			"sisiutl",
-			"campacti",
-			"stench blossom",
-			"lord of the wyverns",
-			"pylraster",
-			"slipkinx steeljoints",
-			"stolas",
-			"agathos",
-			"bune"
+			["coerthas western highlands", "lyuba", "mirka"],
```

Comment:
this data shape hurts my brain.


but c# generally does that as well so i guess it's business as usual